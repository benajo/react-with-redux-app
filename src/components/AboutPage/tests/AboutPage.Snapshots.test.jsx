import React from 'react';
import renderer from 'react-test-renderer';
import AboutPage from '../AboutPage';

it('render AboutPage component', () => {
  const tree = renderer.create(<AboutPage />);
  expect(tree).toMatchSnapshot();
});
