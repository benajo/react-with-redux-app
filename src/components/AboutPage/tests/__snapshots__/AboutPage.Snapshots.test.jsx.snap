// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render AboutPage component 1`] = `
<div>
  <h2>
    About Us
  </h2>
  <p>
    This app uses React, Redux, React Router, and many other helpful libraries.
  </p>
</div>
`;
